Solutions/CohesitySecurity/Tests/helios.py

```diff
@@ -96,32 +96,44 @@ def get_days_ago_timestamp(days_ago):
     return [jsObj["id"] for jsObj in response.json()]
 
 
-def get_recoveries(cluster_id: str, api_key: str) -> dict:
+def get_recoveries(cluster_id: str, api_key: str, start_time_usecs=None, end_time_usecs=None) -> dict:
```

Comment:
Format this file so that the line limit is honored?

Reply:
done